test/test_automl/test_automl.py

```diff
@@ -563,6 +563,10 @@ class MyException(Exception):
     logger = logging.getLogger(logger_name)
     logfile = os.path.join(backend.temporary_directory, logger_name + '.log')
     assert os.path.exists(logfile), print_debug_information(automl) + str(automl._clean_logger())
+
+    # Give some time for the error message to be printed in the
+    # log file
+    time.sleep(3)
```

Comment:
Hey, it would be even safer to check the output below in a for-loop with for example 10 repetitions and with an exponentially increasing sleep, what do you think?

Reply:
Thanks! I have added this recommendation with 5 seconds. If it takes more than this I for sure want it to fail as that is unacceptably long :)